packages/fxa-content-server/server/lib/routes/post-third-party-auth-redirect.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+'use strict';
+
+const { URLSearchParams } = require('url');
+
+module.exports = function () {
+  return {
+    method: 'post',
+    path: '/post_verify/third_party_auth/callback',
+    process: (req, res) => {
+      const urlParameters = new URLSearchParams(req.body);
+      urlParameters.set('provider', 'apple');
```

Comment:
I guess content server doesn't handle TS at all😭

Is this only hit for Apple providers?

Reply:
I'll add some comments here about what is going on.